democracy_club/templates/_includes/voter_info_documentation.html

```diff
@@ -0,0 +1,107 @@
+<ul class="documentation-list">
+    <div class="ds-stack-smaller ds-grid">
+        Elections covered
+        <li>
```

Comment:
This still isn't quite right...

* `div` can't go directly inside a `ul`, so I think this div should be removed and the classes applied to the `ul` directly.`
* The text (here and in the below) needs to be inside the `li`, so we have `ul->li->[text] <ul>`
* I think if you remove `ds-grid` and just use `documentation-list` it looks better, but I remember we talked about this, so it might not for you?